### DISCUSSION TITLE
For some reason, I would like to change Redux to Zustand or Jotai, What should I do?

### DISCUSSION THREAD

**Discussion by nikbobo** [pmndrs/zustand#2107] [General]
Hi, I am a beginners in front-end development.
For some reason, I would like to change Redux to Zustand or Jotai, and remove redux in my project anywhere.
My Project just using `useSelector` `useDispatch` `createStore`  `Provider` four api with redux.
For the better, which will move faster and easiler? Zustand or Jotai? 
And could you give me some suggestion? Could change it by simpily replace api or function?

**Comment by dbritto-dev** [comment]
@nikbobo since you are using `redux` the natural replacement is `zustand`, using `jotai` would be a like a refactor when you change everything. BTW, I would consider changing from `redux` to `zustand` like an enhancement rather than a `refactor`.
